app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -82,18 +82,20 @@ def feeds
     end
 
     def machine_id(feed)
-      os_resource_for(feed).properties['Machine Id']
+      os_resource_for(feed).try(:properties).try { |prop| prop['Machine Id'] }
     end
 
```

Comment:
During the refresh [in](https://github.com/ManageIQ/manageiq/blob/8bd56f34a34519c2b405edc82a7311b0946d9127/app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb#L33) `middleware_manager/refresh_parser.rb` + in specs